bears/vcs/git/GitCommitBear.py

```diff
@@ -241,8 +241,9 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
-        if body_regex and not re.search(body_regex, body):
-            yield Result(self, 'No match found in commit message for the '
+        if body_regex and not re.fullmatch(
+                body_regex, body.strip(), re.MULTILINE | re.DOTALL):
```

Comment:
I wouldn't enable `re.MULTILINE` and `re.DOTALL`, normally people don't expect this + this is not necessary^^